samples/good-arc.rb

```diff
@@ -26,8 +26,8 @@
       10.times do |j|
         strokewidth sw * j
         size = (basesize + step * j) / 3
-        top = (self.height - size) / 2
-        left = (self.width - size) / 2
+        top = (height - size) / 2
+        left = (width - size) / 2
```

Comment:
Ran this and it looks good to me. Also ran a few of the other samples modified in this commit and they all seem ok.